main.nf

```diff
@@ -0,0 +1,179 @@
+
+        Usage:
+         The basic command for running the pipeline is as follows:
+
+         nextflow run main.nf --assembly contigs.fasta
+
+         Mandatory arguments:
+          --assembly                      (Meta)genomic assembly in fasta format (uncompress)
+
+         Optional arguments:
+    ** Extra annotations provided by the user
+        * Genes prediction  
+          --user_genes                    Use the user annotation files. See --prot_gff [false]
+          --prot_gff                      Annotation file in GFF3 format. Mandatory with '--user_genes true'
+        * VIRify (v2.0)
+          --virify                        Integrate VIRify v2.0 results to geNomad predictions [false]
+          --vir_gff                       The final result of VIRify on gff format (08-final/gff/*.gff). Mandatory with '--virify true'
+          --vir_checkv                    CheckV results generated by VIRify (07-checkv/*quality_summary.tsv). Mandatory with '--virify true'
+        * PaliDIS (v2.3.4)
+          --palidis                       Incorporate PaliDIS predictions to final output [false]
+          --palidis_info                  Information file of PaliDIS insertion sequences. Mandatory with '--palidis true'
+    ** Extra annotations inside the pipeline
+          --skip_crispr                   Not to run CRISPRCasFinder. Default behaviour is to run it [false]
+          --skip_amr                      Not to run AMRFinderPlus. Default behaviour is to run it [false]
+    ** Final output validation
+          --gff_validation                Validation of the GFF3 mobilome output [true]
+    ** Output directory
+          --outdir                        Output directory to place results [mobilome_results]
+    ** Show usage message and exit
+          --help                          This usage statement [false]
+        """
+}
+
+if (params.help) {
+	helpMessage()
+	exit 0
+}
+
+
+/*
+    ~~~~~~~~~~~~~~~~~~~~~~~
+        Default options
+    ~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+params.user_genes = false
+params.virify = false
+params.palidis = false
+params.gff_validation = true
+params.outdir = 'mobilome_results'
+params.skip_crispr = false
+params.skip_amr = false
+
+
+/*
+    ~~~~~~~~~~~~~~~~~~~~
+        Run workflow
+    ~~~~~~~~~~~~~~~~~~~~
+*/
+
+workflow {
+	assembly = Channel.fromPath( params.assembly, checkIfExists: true )
+	
+	//PREPROCESSING
+	RENAME( assembly )
+	PROKKA( RENAME.out.contigs_1kb )
+
+
+	// PREDICTION
+	GENOMAD( RENAME.out.contigs_5kb )
+	GBK_SPLITTER( PROKKA.out.prokka_gbk )
+	ICEFINDER( 
+		GBK_SPLITTER.out.gbks,
+		file( "${params.outdir}/prediction/icefinder_results/gbk"),
```

Comment:
add the `checkIfExists` parameter to avoid errors downstream

Reply:
Done